ocsp/ocsp.go

```diff
@@ -164,8 +164,10 @@ func (s StandardSigner) Sign(req SignRequest) ([]byte, error) {
 	if bytes.Compare(req.Certificate.RawIssuer, s.issuer.RawSubject) != 0 {
 		return nil, cferr.New(cferr.OCSPError, cferr.IssuerMismatch)
 	}
-	if req.Certificate.CheckSignatureFrom(s.issuer) != nil {
-		return nil, cferr.New(cferr.OCSPError, cferr.IssuerMismatch)
+
+	err := req.Certificate.CheckSignatureFrom(s.issuer)
+	if err != nil {
```

Comment:
The should probably be

```
return nil, cferr.New(cferr.CertificateError, cferr.VerifyFailed, err)
```

Reply:
Sorry, it's the tests point out that it's cferr.Wrap, not cferr.New.

Reply:
No worries, i saw the build failure but had not had a chance to test it locally before I saw it. I wasnt thinking about PR refresh when I made the change from github directly ;). Update will be forthcoming.

Reply:
I went with cferr.OCSPError instead of CertificateError given the pattern in the rest of this function. I think it makes sense to me, context of OCSP signing, Verify failed, then verify real error.

Reply:
So I just double checked something:

* https://github.com/cloudflare/cfssl/blob/master/errors/error.go#L417 `CertificateError` will cause it to add more information to the error code (maybe not useful here).
* When we added the `OCSPError`, we didn't add it to [the default case](https://github.com/cloudflare/cfssl/blob/master/errors/error.go#L428). So, OCSP error will make cfssl panic right now.

I'll merge this, then submit a PR to fix the default case.